src/appleseed/renderer/modeling/bsdf/metalbrdf.cpp

```diff
@@ -77,13 +77,8 @@ namespace
     //   [1] Microfacet Models for Refraction through Rough Surfaces
     //       http://www.cs.cornell.edu/~srm/publications/EGSR07-btdf.pdf
     //
-    // Todo:
-    //
-    //  This BRDF currently assumes an air | metal interface and will not
```

Comment:
Wasn't this comment helpful?

Reply:
It is outdated. We removed this limitation some time ago.

Reply:
Cool.